### PR TITLE
Update gpcrondump help file

### PR DESCRIPTION
**Pull request by Chibin**
* The user can now update the DDBOOST_CONFIG storage unit,
  which needs to be reflected on the help file.

### CODE REVIEW

**Comment by mkiyama** [comment]
Here are my edits.  the number (e.g. 253) are the line numbers.

253
 --ddboost-storage-unit is optional. Specify a Data Domain system storage
 unit ID where a back up is stored. The storage unit ID is also used 
 to replicate dump files between source and target Data Domain systems
 if --ddboost-remote is specified. 
 
 If this option is not specified, the default is the value in
 the DD Boost config file for Greenplum Database cluster.


331
 --ddboost-remote is optional. Indicates that the configuration 
 parameters are for the remote Data Domain system that is used for backup 
 replication Data Domain Boost managed file replication. 
 
 If --ddboost-remote is specified, --ddboost-storage-unit is ignored.


367
--ddboost-show-config [--ddboost-remote]
 
 Show DDBoost and MFR (--ddboost-remote) related configuration 
 information: DDBoost hostname, username, default backup directory,
 default storage unit.

**Comment by Chibin** [comment]
Updated with the comments above.

**Comment by ascherbaum-pivotal** [comment]
Looks good, let's get this committed.

File: gpMgmt/doc/gpcrondump_help [hunk 1]
```diff
@@ -249,11 +250,13 @@ OPTIONS
  set up the Data Domain Boost credential, as described in the next option 
  below. 
 
- --ddboost-storage-unit is optional, which provides a dynamic storage
- unit where to replicate dump files between source and target DDBoost Servers,
- backup directory defaults to the cluster's config file when the Data Domain
- Boost credentials were set. If this option is not specified, it also defaults
- to the cluster's config file.
+ --ddboost-storage-unit is optional. Specify a Data Domain system storage
+ unit ID where a back up is stored. The storage unit ID is also used
```
Comment (kaknikhil) [inline gpMgmt/doc/gpcrondump_help]: should it be storage unit name instead of ID?

File: gpMgmt/doc/gpcrondump_help [hunk 2]
```diff
@@ -249,11 +250,13 @@ OPTIONS
...
+ to replicate dump files between source and target Data Domain systems
+ if --ddboost-remote is specified.
+
+ If this option is not specified, the default is the value in
```
Comment (kaknikhil) [inline gpMgmt/doc/gpcrondump_help]: Can we rephrase this as follows:
"If --ddboost-storage-unit is not specified,  it defaults to the value stored in the DD Boost config file for the Greenplum Database cluster" .
